index.html

```diff
@@ -1,48 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>T8 Character Moves Rant</title>
-    <!-- <meta name="description"> T8 Moves Rant/Guide </meta> -->
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width" />
-    <!--
+
+<head>
+  <title>T8 Character Moves Rant</title>
+  <!-- <meta name="description"> T8 Moves Rant/Guide </meta> -->

```

Comment:
to fix this line, change it to 
`<meta name="description" content="T8 Moves Rant/Guide" />`
(see documentation [here](https://www.w3schools.com/tags/tag_meta.asp))